packages/sdk/observability/src/helpers/common.ts

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright 2022 DXOS.org
+//
+
+import type { Client } from '@dxos/client';
+
+// TODO(wittjosiah): Improve privacy of telemetry identifiers.
+//  - Identifier should be generated client-side with no attachment to identity.
+//  - Identifier can then be reset by user.
+//  - Identifier can be synced via HALO to allow for correlation of events bewteen devices.
+//  - Identifier should also be stored outside of HALO such that it is available immediately on startup.
+export const getTelemetryIdentifier = (client: Client) => {
+  if (!client?.initialized) {
+    return undefined;
+  }
+
+  const identity = client.halo.identity.get();
+  if (identity) {
+    return identity.identityKey.toHex();
```

Comment:
```suggestion
    return identity.identityKey.truncate();
```
to align with other places we're sending this?

Reply:
whoops, must have missed this in the merge